services/ux-backend/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.20 as builder
+
+WORKDIR /go/src/github.com/red-hat-storage/ocs-operator
+COPY . .
+USER root
+
+RUN make build-go
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal
+
+COPY --from=builder /go/src/github.com/red-hat-storage/ocs-operator/build/_output/bin/ux-backend-server /usr/local/bin/ux-backend-server
+
+ENTRYPOINT ["/usr/local/bin/ux-backend-server"]
+
```

```suggestion
USER operator
```
- pls follow conventions from other Dockerfiles in the repo